app/monitoringapi.go

```diff
@@ -129,11 +135,14 @@ func startReadyChecker(ctx context.Context, tcpNode host.Host, eth2Cl eth2wrap.C
 					log.Warn(ctx, "Failed to get beacon node peer count", bnErr)
 				}
 
-				syncing, err := beaconNodeSyncing(ctx, eth2Cl)
+				syncing, syncDistance, err := beaconNodeSyncing(ctx, eth2Cl)
 				//nolint:nestif
 				if err != nil {
 					err = errReadyBeaconNodeDown
 					readyzGauge.Set(readyzBeaconNodeDown)
+				} else if syncDistance > bnFarBehindSlots {
```

Comment:
okay, then this would be the flow of execution:
```
readyzBeaconNodeDown
readyzBeaconNodeZeroPeers
readyzBeaconNodeFarBehind
readyzBeaconNodeSyncing
```

Reply:
Gotcha. This is the updated flow now:
```
readyzBeaconNodeDown
readyzBeaconNodeSyncing
readyzBeaconNodeZeroPeers
readyzBeaconNodeFarBehind
```